rest/data/interface.go

```diff
@@ -54,348 +35,10 @@ type Connector interface {
-	// FindPodByID finds a pod by the given ID.
-	FindPodByID(id string) (*restModel.APIPod, error)
-	// FindPodByExternalID finds a pod by the given external identifier.
-	FindPodByExternalID(id string) (*restModel.APIPod, error)
-	// UpdatePodStatus updates a pod's status by ID.
-	UpdatePodStatus(id string, current, updated restModel.APIPodStatus) error
-	// CheckPodSecret checks that the ID and secret match the server's
-	// stored credentials for the pod.
-	CheckPodSecret(id, secret string) error
-
-	// FindProjectAliases queries the database to find all aliases (including or excluding those specified).
-	// Includes repo aliases if repoId is provided.
-	FindProjectAliases(string, string, []restModel.APIProjectAlias) ([]restModel.APIProjectAlias, error)
-	// CopyProjectAliases copies aliases from the first project for the second project.
-	CopyProjectAliases(string, string) error
-	// UpdateProjectAliases upserts/deletes aliases for the given project
-	UpdateProjectAliases(string, []restModel.APIProjectAlias) error
-	// UpdateAliasesForSection, given a project, a list of current aliases, a list of previous aliases, and a project page section,
-	// upserts any current aliases, and deletes any aliases that existed previously but not anymore (only
-	// considers the aliases that are relevant for the section). Returns if any aliases have been modified.
-	UpdateAliasesForSection(string, []restModel.APIProjectAlias, []model.ProjectAlias, model.ProjectPageSection) (bool, error)
-	// HasMatchingGitTagAliasAndRemotePath returns true if the project has aliases defined that match the given tag, and
-	// returns the remote path if applicable
-	HasMatchingGitTagAliasAndRemotePath(string, string) (bool, string, error)
-	// TriggerRepotracker creates an amboy job to get the commits from a
-	// Github Push Event
-	TriggerRepotracker(amboy.Queue, string, *github.PushEvent) error
-
-	// GetCLIUpdate fetches the current cli version and the urls to download
-	GetCLIUpdate() (*restModel.APICLIUpdate, error)
-
-	// GenerateTasks parses JSON files for `generate.tasks` and creates the new builds and tasks.
-	GenerateTasks(context.Context, string, []json.RawMessage, amboy.QueueGroup) error
-
-	// GeneratePoll checks to see if a `generate.tasks` job has finished.
-	GeneratePoll(context.Context, string, amboy.QueueGroup) (bool, []string, error)
-
-	// SaveSubscriptions, given an owner, a list of current subscriptions and an isProject boolean and
-	// upserts the given set of API subscriptions (if the owner is a project type, verify that all current subscriptions have
-	// the right owner and owner type).
-	SaveSubscriptions(string, []restModel.APISubscription, bool) error
-	// GetSubscriptions returns the subscriptions that belong to a user
-	GetSubscriptions(string, event.OwnerType) ([]restModel.APISubscription, error)
-	DeleteSubscriptions(string, []string) error
-	// CopyProjectSubscriptions copies subscriptions from the first project for the second project.
```

Comment:
Can you move this comment (and others if needed) to be function comments?

Reply:
I thought I had caught all of these but I suppose not and missed a few - moved the remaining comments